client/reader/controller.js

```diff
@@ -292,7 +293,7 @@ module.exports = {
 						analyticsPageTitle,
 						mcKey
 					),
-					showPrimaryFollowButtonOnCards: false,
+					showPrimaryFollowButtonOnCards: true,
```

Comment:
shouldn't this be false, everything in the a8c stream is from sites you already follow